packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -121,6 +121,8 @@ export interface CallArrangementProps {
   setIsPromptOpen?: (isOpen: boolean) => void;
   /* @conditional-compile-remove(spotlight) */
   setPromptProps?: (props: PromptProps) => void;
+  /* @conditional-compile-remove(spotlight) */
+  hideSpotlightFeature?: boolean;
```

Comment:
I think the naming here might imply that the local user loses ALL controls to spotlight maybe we should try to find a name for these variables across the board that allows the end user to still un-spotlight themselves